src/data-fetching/api/PrezlyApi.ts

```diff
@@ -75,7 +76,8 @@ export class PrezlyApi {
         }
 
         try {
-            return await this.sdk.stories.get(uuid);
+            const story = await this.sdk.stories.get(uuid);
+            return story;
```

Comment:
For some reason i have error from eslint when i try to just return awaited function. call
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/3620639/206244056-5463e4bd-e537-4cd2-8015-8e1ddac11230.png">

Reply:
I think this is fine, it's making the code a bit more readable IMO

Reply:
Why not just do ` return this.sdk.stories.get(uuid)`? That's what ESLint wants, right?